Package.swift

```diff
@@ -29,15 +31,19 @@ let package = Package(
     targets: [
         .target(
             name: "Benchmark",
-            dependencies: [.product(name: "ArgumentParser", package: "swift-argument-parser")]),
+            dependencies: [.product(name: "ArgumentParser", package: "swift-argument-parser")],
+            swiftSettings: optimize),
```

Comment:
Hmmm, by always setting those flags on this repo, we'll be making it more difficult to debug the library. Could you explain why this is necessary?

Reply:
It was a shortcut to force cross module optimization, without having to specify it manually when doing `swift run`. It was used for perf debugging, and wasn't intended to be a part of the PR.